examples/tpm2_filternv/tpm2_filternv

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python3
+
+import argparse
+import os
+from binascii import hexlify
+from tpm2_pytss.ESAPI import ESAPI
+from tpm2_pytss.TCTILdr import TCTILdr
+from tpm2_pytss.constants import TPMA_NV, TPM2_CAP, TPM2_HC
+
+
+output_template = """{pub.nvIndex:#x}:
+  name: {hexname}
+  hash algorithm:
+    friendly: {pub.nameAlg}
+    value: {pub.nameAlg:#x}
+  attributes:
+    friendly: {pub.attributes}
+    value: {pub.attributes:#x}
+  size: {pub.dataSize}
+"""
+
+
+def get_tcti(arg):
+    if arg:
+        return TCTILdr.parse(arg)
+    elif os.getenv("PYTSS_EXAMPLE_TCTI"):
+        return TCTILdr.parse(os.getenv("PYTSS_EXAMPLE_TCTI"))
+    return TCTILdr()
+
+
+def tpma_nv_arg(arg):
+    try:
+        return TPMA_NV(arg, 0)
+    except:
```

Comment:
In some other projects, I saw a generic Python recommendation of avoiding the use of bare `except:` statements. Here it seems that the only exception which is expected to happen is `ValueError`. Is it possible to use `except ValueError:` instead?

By the way, the `TPMA_NV(arg, 0)` syntax is difficult to understand for newcomers who can wonder: what is this `0`? Is this syntax documented somewhere? In my humble opinion it is clearer to use `TPMA_NV(int(arg, 0))`, or if you want to about this call to `int()`, something like `TPMA_NV(arg, base=0)` to hint that the constructor of `int` is invoked here. What do you think?

Reply:
good points, fixed